Gruntfile.js

```diff
@@ -0,0 +1,145 @@
+module.exports = function(grunt) {
+    grunt.initConfig({
+        concurrent: {
```

Comment:
I don't think I'm using `grunt-concurrent` anymore since I figured out how to start the server via the spec files themselves. I can remove this.